docs/dev-guide/contributing.md

```diff
@@ -7,13 +7,19 @@ title: Contributing Guidelines
 We would love to have your help. Before you start working however, please read
 and follow this short guide.
 
-# Reporting Issues
-Provide as much information as possible. Mention the version of Jitsi Meet, 
-Jicofo and JVB you are using, and explain (as detailed as you can) how the 
-problem can be reproduced.
+# Reporting Issues and Asking Questions
+
+We prefer issues to be discussed first in the [community forum](https://community.jitsi.org/) and when confirmed, then an issue can be opened in the [Issue tracker](https://github.com/jitsi/jitsi-meet/issues).
+
+When opening an issue, please provide as much information as possible. Mention the version of Jitsi Meet, Jicofo and JVB you are using, and explain (as detailed as you can) how the problem can be reproduced.
+
+Please ask any general and implementation specific questions on the [community forum](https://community.jitsi.org/) for support.
 
 # Code contributions
-Found a bug and know how to fix it? Great! Please read on.
+
+Visit the [Issue tracker](https://github.com/jitsi/jitsi-meet/issues) to find a list of open issues that need attention.
+
+Our [Developer Guide](dev-guide.md) will help you to setup a development environment to start working on the Jitsi Meet applications.
```

Comment:
Have you tested this type of linking when rebuilding the site with docusaurus?

Reply:
Yeah I tested the links by building locally, i.e. npm start.

The external links open in a new tab, and the Developer Guide link opens in the same tab.